.ruff.toml

```diff
@@ -238,18 +237,23 @@ lint.unfixable = [
     "PTH114",  # os-path-islink
     "PTH206",  # Replace `.split(os.sep)` with `Path.parts`
     "RET506",  # superfluous-else-raise
+    "RET505",  # superfluous-else-return
+
 ]
 "astropy/constants/*" = ["N817"]  # camelcase-imported-as-acronym
 "astropy/convolution/*" = [
     "RET506",  # superfluous-else-raise
+    "RET505",  # superfluous-else-return
 ]
 "astropy/coordinates/*" = [
     "DTZ007",  # call-datetime-strptime-without-zone
+    "RET505",  # superfluous-else-return
 ]
 "astropy/cosmology/*" = [
     "C408",  # unnecessary-collection-call
     "PT019",   # pytest-fixture-param-without-value
     "RET506",  # superfluous-else-raise
+    "RET505",  # superfluous-else-return
```

Comment:
It's also best to keep the package ignores alphabetically sorted. The strategy of always adding new ignores to the ends of the lists maximizes the number of merge conflicts in case of concurrent pull requests.